src/api.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * PNG-parts
+ * parts of a Portable Network Graphics implementation
+ * Copyright 2018 Cody Licorish
+ *
+ * Licensed under the MIT License.
+ *
+ * api.h
+ * API main header
+ */
+#ifndef __PNG_PARTS_API_H__
+#define __PNG_PARTS_API_H__
+
+/*
+ * export variable
+ */
+#ifndef pngparts_api
+#define pngparts_api
```

Comment:
The file has the `pngparts_api` variable, which is actually a macro.
The macro should be capitalized first.